src/renderers/testing/ReactTestRenderer.js

```diff
@@ -60,12 +62,12 @@ ReactTestComponent.prototype.receiveComponent = function(
   this._currentElement = nextElement;
   this.updateChildren(nextElement.props.children, transaction, context);
 };
+
 ReactTestComponent.prototype.getHostNode = function() {};
-ReactTestComponent.prototype.getPublicInstance = function() {
-  // I can't say this makes a ton of sense but it seems better than throwing.
-  // Maybe we'll revise later if someone has a good use case.
-  return null;
+ReactTestComponent.prototype.getPublicInstance = function(transaction) {
```

Comment:
Let's add a newline before this line too for consistency now that we’ve put some of them between methods.